src/nupic/engine/Link.hpp

```diff
@@ -394,70 +394,6 @@ namespace nupic
     using Serializable::read;
     void read(LinkProto::Reader& proto);
 
-
```

Comment:
None of these are implemented and the declarations sans-implementation cause symbol errors at runtime in python.